src/v2/ion/responseTransformer.js

```diff
@@ -230,8 +249,9 @@ const convert = (settings, idx = {}, lastResult = null) => {
 
   injectIdPConfigButtonToRemediation(settings, result);
   if (!result.messages) {
-    // Only redirect to the IdP if we are not in an error flow
+    // Only redirect to the IdP or Device Trust Mtls if we are not in an error flow
```

Comment:
Is it possible that there is both a `redirect-device-trust` and `redirect-idp` remediation form? If so would that cause any problems? Or is it fine that `convertRedirectToSuccessRedirectForMtls` would just override result.remediations with the Mtls href?

Reply:
It should never happen. IDP routing happens at the Identify Step. Device Trust MTLS redirect happens post identify step after the user is identified. Also, I added the check `result.remediations.length === 1` to make sure to redirect to DT one only when it is the only remediation in the list

Reply:
Accidentally glossed over the `result.remediations.length === 1`, makes sense!